pkg/pipelines/bootstrap.go

```diff
@@ -107,6 +111,7 @@ func bootstrapResources(o *BootstrapOptions, appFs afero.Fs) (res.Resources, err
 	appName := repoToAppName(repoName)
 	serviceName := repoName
 	secretName := secrets.MakeServiceWebhookSecretName(ns["dev"], serviceName)
+	logBootstrapStatus("Bootstrapping dev and stage environment resources")
```

Comment:
We can't hardcode environment names  because of prefix

Reply:
This has been addressed by taking into consideration the prefix.